pkg/reconciler/revision/resources/queue.go

```diff
@@ -224,12 +224,13 @@ func makeQueueContainer(rev *v1alpha1.Revision, loggingConfig *logging.Config, t
 		return nil, errors.Wrap(err, "failed to serialize readiness probe")
 	}
 
+	servingPort := strconv.Itoa(int(ports[len(ports)-1].ContainerPort))
```

Comment:
This port can be statically decided. It's either `queueHTTPPort` or `queueHTTP2Port`. You can modify the respective `if` above to set `servingPort` and then append `servingPort` to the others.

Reply:
Done.